src/Util/PipVersion.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Php\Pie\Util;
+
+use Composer\InstalledVersions;
+
+use function class_exists;
+
+class PipVersion
```

Comment:
Typo in class and filename, should be `PieVersion`. Should also be `final` as this should not be overridden:

```suggestion
final class PieVersion
```

Reply:
nice catch, updated, @asgrim PTAL,